README.md

```diff
@@ -0,0 +1,28 @@
+<h4> 1. The purpose of clisnt-libs-java-samples </h4>
+  
+All examples shown below fails, because samples does not contain appropriate adscore signature
+and key. These samples show usage of the Signature Verify API, and should be used with correct
+data.
+   
+<h4>2. Which classes are samples?</h4>
+All classes that implement Sample.java interface contain sample code.
```

Comment:
we also should provide requirements to run sample, i.e. jdk XX

Reply:
Added!